config/manifests-crd/bases/recipe.clusterserviceversion.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: recipe.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Recipe is the Schema for the recipes API
+      displayName: Recipe
+      kind: Recipe
+      name: recipes.ramendr.openshift.io
+      version: v1alpha1
+  description: Recipe description. TODO.
```

Comment:
We can use: "Recipe describes a workflow used for a capturing or recovering Kubernetes resources."

(based on https://github.com/RamenDR/ramen/blob/main/docs/recipe.md)